rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -101,28 +108,22 @@ def _compute_necessary_resources(self):
         return {"num_workers": num_workers, "use_gpu": bool(num_gpus)}
 
     def update(self, batch: MultiAgentBatch = None, **kwargs):
-        """TODO: account for **kwargs
-        Example in DQN:
+        """
+        Example:
             >>> trainer_runner.update(batch) # updates the gradient
-            >>> trainer_runner.update(update_target=True) # should soft-update
-                the target network
         """
         refs = []
-        if batch is None:
-            for worker in self.workers:
-                refs.append(worker.update.remote(**kwargs))
-        else:
-            global_size = len(self.workers)
-            batch_size = math.ceil(len(batch) / global_size)
-            for i, worker in enumerate(self.workers):
-                batch_to_send = {}
-                for pid, sub_batch in batch.policy_batches.items():
-                    batch_size = math.ceil(len(sub_batch) / global_size)
-                    start = batch_size * i
-                    end = min(start + batch_size, len(sub_batch))
-                    batch_to_send[pid] = sub_batch[int(start) : int(end)]
-                new_batch = MultiAgentBatch(batch_to_send, int(batch_size))
-                refs.append(worker.update.remote(new_batch))
+        global_size = len(self.workers)
+        batch_size = math.ceil(len(batch) / global_size)
```

Comment:
why do we need to calculate this size here, unused, then immediately get updated below?